src/integ_test/java/games/strategy/engine/lobby/server/userDB/DbUserControllerIntegrationTest.java

```diff
@@ -15,90 +16,118 @@
 
 public class DbUserControllerIntegrationTest {
 
+
+
   @Test
-  public void testCreate() throws Exception {
+  public void create() throws Exception {
+    testCreate(Database::getDerbyConnection, new DbUserController());
+  }
+
+  private void testCreate(Supplier<Connection> connectionSupplier, DbUserController controller) throws Exception {
```

Comment:
@DanVanAtta For your next PR...  I just noticed that the `create()` and `testCreate()` method names should probably be swapped.